src/gui/widgets/TrackContentWidget.cpp

```diff
@@ -572,8 +572,7 @@ void TrackContentWidget::mousePressEvent( QMouseEvent * me )
 			so.at( i )->setSelected( false);
 		}
 		getTrack()->addJournalCheckPoint();
-		const TimePos pos = getPosition( me->x() ).getBar() *
-						TimePos::ticksPerBar();
+		const TimePos pos = getPosition( me->x() ).getTicks();
```

Comment:
To preserve old behavior(not noticeable unless cloning tracks inside BB to song editor, though):
```suggestion
		const TimePos pos = getPosition( me->x() ).getTicks() / DefaultTicksPerBar * DefaultTicksPerBar;
```